fastai/tabular/model.py

```diff
@@ -20,16 +22,30 @@ def _one_emb_sz(classes, n, sz_dict=None):
     return n_cat,sz
 
 # Cell
-def get_emb_sz(to, sz_dict=None):
-    "Get default embedding size from `TabularPreprocessor` `proc` or the ones in `sz_dict`"
+def get_emb_sz(
+    to:(Tabular, TabularPandas), # Tabular or TabularPandas object
+    sz_dict:dict=None # Dict containing embedding sizes to override default emb_sz_rule (i.e. {'catclass1':100})
```

Comment:
```suggestion
    sz_dict:dict=None # Dictionary of {key : size} to override default `emb_sz_rule`
```

Reply:
I still like having a bit more of an example here than `{key : size}` 
```suggestion
    sz_dict:dict=None # Dictionary of {'class_name' : size,...} to override default `emb_sz_rule` 
```